docs/community/contribute/compliance.md

```diff
@@ -0,0 +1,151 @@
+# Contribute Compliance Checks in Trivy
+
+Trivy supports several different compliance checks. The details on compliance scanning with Trivy are provided in the [compliance documentation](../../docs/compliance/compliance.md).
+All of the Compliance Checks currently available in Trivy can be found in the `trivy-policies/specs/compliance/` directory ([Link](https://github.com/aquasecurity/trivy-policies/tree/main/specs/compliance)).
+
+New checks are based on the custom compliance report detailed in the [main documentation.](../../docs/compliance/compliance/#custom-compliance)
+If you would like to create your custom compliance report, please reference the information in the main documentation. This section details how community members can contribute new Compliance Specs to Trivy.
+
+All compliance specs in Trivy are based on the formal compliance reports such as CIS Benchmarks.
+
+## Contributing new Compliance Specs
+
+Compliance specs can be based on new compliance reports becoming available, or identifying missing compliance specs that Trivy users would like to access. 
+
+### Create a new Compliance Spec
+
+The existing compliance specs in Trivy are located under the `trivy-policies/specs/compliance/` directory ([Link](https://github.com/aquasecurity/trivy-policies/tree/main/specs/compliance)).
+
+Create a new file under `trivy-policies/specs/compliance/` and name the file in the format of "provider-resource-spectype-version.yaml". For example, the file name for AWS CIS Benchmarks for EKS version 1.4 is: `aws-eks-cis-1.4.yaml`.
```

Comment:
it's a good idea to define a naming convention. 
"provider-resource" not sure this is the best terminology, also very cloud specific (and the compliance feature is not). I think in this case the entire "aws-eks" is the subject and there's no need to specify it's components in the generic template. another example there will be CIS specs for specific k8s flavor (like open shift), which this needs to appear in the name but I don't think in the template. 
so I would suggest to simplify the template as `subject-spec.yaml` which means: specify the subject of the compliance (what are we evaluating?). if it's a cloud service or another example of a subcomponent, include the parent too, for example `aws-eks`. if the spec is specific to a version, optionally include it after the subject, e.g `k8s-1.21`. spec is a short name for the compliance framework that people can identify, e.g `cis`. if the spec is versioned, include the version too `cis-1.1`. wdyt?

Reply:
What do you mean with this sentence:
> which this needs to appear in the name but I don't think in the template.

"provider-resource-spectype-version.yaml" the reasoning here was is that we call it provider and resource/service in other parts of the source code and docs 

> if it's a cloud service or another example of a subcomponent, include the parent too, for example aws-eks

The spec type is specific to a benchmark version e.g. CIS, I don't think it makes much sense to include the K8s version as that won't tell people more about the compliance scan and in some cases CIS 
specs correspond to different k8s versions @chen-keinan thoughts?

recent CIS specs for reference 
<img width="481" alt="Screenshot 2024-04-02 at 15 42 58" src="https://github.com/aquasecurity/trivy/assets/33576047/065546b4-8284-4ff7-8db0-b285c3dd1a35">

Reply:
> What do you mean with this sentence

For example, that opehshift needs to appear in the file name, but doesn't have to have a placeholder in the naming convention template

> we call it provider and resource/service in other parts of the source code and docs

I think this is true only to cloud. for example, I wrote a k8s compliance (PSS) and didn't have to know about service/provider in trivy and didn't use it in the name

> I don't think it makes much sense to include the K8s version

OK, makes sense. but just to clarify, this isn't what text you quoted says "include the parent too, for example aws-eks"